packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/index.tsx

```diff
@@ -43,6 +62,16 @@ export const RainbowKitCustomConnectButton = () => {
 
               return (
                 <>
+                  <div className="flex flex-row items-center gap-x-4 mr-1">
+                    <p>
+                      Member Status:{" "}
+                      {isMember ? "✅" : "❌"}
+                    </p>
+                    <p>
+                      Check-in Status:{" "}
+                    {isCheckedIn ? "✅" : "❌"}
+                    </p>
+                  </div>
```

Comment:
This is taking up a lot of space, which means it will also draw a lot of attention from the user.  

Consider making it smaller, maybe a single icon that changes its image and/or color depending on the three different statuses:  
- Not a member  
- Member with no check-in  
- Checked-in member